src/inc/crsttypes.h

```diff
@@ -1,6 +1,8 @@
+//
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
+//
```

Comment:
I shouldn't have looked at the diff a little closer. You meant the extra // at the beginning and end. I'll edit them out.